src/packages/core/workspace/workspace-property-layout/workspace-property-layout.element.ts

```diff
@@ -102,7 +118,22 @@ export class UmbWorkspacePropertyLayoutElement extends LitElement {
 				position: sticky;
 				top: var(--uui-size-space-4);
 				height: min-content;
-				z-index: 2;
+				z-index: 1050;
```

Comment:
May I ask why you want this super high z-index number?

I know its typical CSS before shadow DOM to do so. But I would really like us to avoid such. So please go with the lowest one that does what we need.

Maybe another time, add a comment to describe the value. Is it because it must go on top of something?

Reply:
Ah, I just realized its because it's sticky in the small-mode/mobile version.

Thats a mistake, we do not want the label to float with. cause that creates al sorts of trouble if the editor it self had an action bar that was sticky.

In the case of the RTE, it would make more sense to have the toolbar following the screen. :-)

So I will get rid of this.